.github/workflows/visual-regression.yml

```diff
@@ -5,6 +5,19 @@ on:
     branches:
       - main
       - develop
+    # Only run if CSS/JS/MD-related files changed.
+    paths:
+      - '.github/workflows/visual-regression.yml'
+      - 'assets/**'
+      - 'packages/**'
```

Comment:
Let's exclude tests here as well.

Reply:
My concern was that _eventually_ we might have our own packages that would warrant running the regression tests, but I guess we can cross that bridge when we come to it 😅

Reply:
I don't mind including `packages/` here if you want, I was just saying we should exclude JS tests the paths 😄